### PR TITLE
Define path with __dir__

### PR DESCRIPTION
**Pull request by DirtyF**
Inspired from https://github.com/jekyll/jekyll/pull/6087 who itself took inspiration from https://github.com/rails/rails/commit/5b8738c2df003a96f0e490c43559747618d10f5f

/cc @bogdanvlviv

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -15,11 +15,11 @@ class Drop
 ENV["JEKYLL_LOG_LEVEL"] = "error"
 
 def dest_dir
-  File.expand_path("../tmp/dest", File.dirname(__FILE__))
+  File.expand_path("tmp/dest", __dir__)
```
Comment (bogdanvlviv) [inline spec/spec_helper.rb]: It should be `File.expand_path("../tmp/dest", __dir__)`

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -15,11 +15,11 @@ class Drop
...
 end
 
 def source_dir
-  File.expand_path("./fixtures", File.dirname(__FILE__))
+  File.expand_path("./fixtures", __dir__)
```
Comment (bogdanvlviv) [inline spec/spec_helper.rb]: would be nice to get rid of redundant `.`

**Comment by benbalter** [comment]
Ooop. Sorry @DirtyF @bogdanvlviv got excited by learning something new and moved way too quickly. Glad to address the comments in a follow up PR. Sorry for the itchy finger. 100% my bad.

**Comment by DirtyF** [comment]
@bogdanvlviv Thanks for the review.
@benbalter As tests were passing, I thought this was OK. 😞

**Comment by benbalter** [comment]
Since it was my mistake, I implemented both changes via cf90f90. Sorry about that.

**Comment by bogdanvlviv** [comment]
@benbalter, @DirtyF Nice work! :heart:

**Pull request merged**